log.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"os"
+	"regexp"
+)
+
+const (
+	// Severity.
+
+	// From /usr/include/sys/syslog.h.
+	// These are the same on Linux, BSD, and OS X.
+	LOG_EMERG int = iota
+	LOG_ALERT
+	LOG_CRIT
+	LOG_ERR
+	LOG_WARNING
+	LOG_NOTICE
+	LOG_INFO
+	LOG_DEBUG
+)
+
+const (
+	EnvLogLevel      = "LOG_LEVEL"
+	EnvLogFormat     = "LOG_FORMAT"
+	defaultLogFormat = "<%s>\t%s" // #1 severity, #2 msg
```

Comment:
I'm using jsonmon in my production with journald integrations. So, I'm not able to merge until it will work again.

Thanks for many interesting ideas, just a couple of minutes ago I switched my base to govendor and many thanks for suggesting it.

Closing this for now, feel free to re-open